addons/website/static/src/builder/plugins/options/parallax_option_plugin.js

```diff
@@ -46,17 +46,30 @@ class WebsiteParallaxPlugin extends Plugin {
         } else {
             delete editingElement.dataset.parallaxType;
         }
-        let parallaxEl = editingElement.querySelector(".s_parallax_bg");
+
+        let parallaxBgEl = editingElement.querySelector(".s_parallax_bg");
+        const parallaxBgWrapEl = editingElement.querySelector(".s_parallax_bg_wrap");
```

Comment:
Should not this search for directly children?

Reply:
@bvr-odoo Was that PR created?

Reply:
> @bvr-odoo Was that PR created?

yes, here https://github.com/odoo/odoo/pull/220979/commits/73aebaddca3aa0ed26f1504b2457583cc6108fc1